### ISSUE TITLE
Having an "@" symbol in the locale JSON file breaks the plugin and results in a [plugin:unplugin-vue-i18n] Invalid linked format (error code: 10)

### ISSUE THREAD

**Issue by rod2ik** [nuxt-modules/i18n#2271]
Hi,

Thanks for your awesome module !

I am still experiencing problems to insert a `@` character into a locale JSON file. I know there has already been an issue about this exact same problem (cf. nuxt-modules/i18n#1519 ), that has been closed since, but I am still experiencing the same problem (sorry about that..).

## Environment

Nuxi 3.6.5

------------------------------
- Operating System: Linux
- Node Version:     v20.3.0
- Nuxt Version:     3.6.5
- Nitro Version:    2.5.2
- Package Manager:  yarn@1.22.19
- Builder:          vite
- User Config:      devtools, modules, i18n
- Runtime Modules:  @nuxtjs/i18n@8.0.0-beta.13
- Build Modules:    -
------------------------------
## Reproduction

Cf [rod2ik/i18n-nuxt](https://github.com/rod2ik/i18n-nuxt) for a playground repo

## Describe the bug

```ERROR  Internal server error: Invalid linked format (error code: 10) in /home/../i18n-nuxt/locales/en.json
  target message: Hello @ World
  target message path: helloWorld.title

  1  |  Hello @ World
   |          ^
```

## Additional context

i18n config in nuxt.config.json :

``` ts
export default defineNuxtConfig({
  devtools: { enabled: true },
  modules: [
    '@nuxtjs/i18n',
  ],
  i18n: {
    /* module options */
    lazy: true,
    langDir: "locales",
    strategy: "no_prefix",
    locales: [
      {
        code: "en",
        iso: "en-US",
        name: "English",
        file: "en.json",
        flag: "🇺🇸"
      },
      {
        code: "fr",
        iso: "fr-FR",
        name: "Français",
        file: "fr.json",
        flag: "🇫🇷"
      },
    ],
    defaultLocale: "en",
  },
})
```

## Logs

``` shell
Vue I18n configuration file does not exist at i18n.config. Skipping...           @nuxtjs/i18n 15:05:37

✔ Nuxt Devtools is enabled v0.7.1 (experimental)                                                    15:05:37

[15:05:38]  ERROR  [unplugin-vue-i18n]  Invalid linked format (error code: 10) in /home/../i18n-nuxt/locales/en.json
  target message: Hello @ World
  target message path: helloWorld.title

  1  |  Hello @ World
   |          ^
```

**Comment by kazupon** [comment]
Thank you for your feedback.

This is not issue, so `@` is special characeters from vue-i18n v9 
nuxt i18n v8 includes vue-i18n v9.
`@` is used as a literal for mesage syntax such as linked message. So you need to use literal interpolation to work around it.
https://vue-i18n.intlify.dev/guide/essentials/syntax.html#special-characters

Thanks!

**Comment by rod2ik** [comment]
Thanks very much for your time and answer. **It reaaally helped me (and I hope others as well)** !
I am afraid google sent me to your `v8` (old) doc pages... 
`v9` docs are perfectly clear.  
It works like a charm, thank you :+1:

**Comment by Asxer** [comment]
It is the problem in my case. I have big translation files that are not built by me. There are emails put in there. This implementation really blocked me. Maybe there is a way to overwrite the special characters list? Downgrading version doesn't help me

**Comment by this-oliver** [comment]
Do you have a recommendation for how to resolve this (i.e. escape the `@` character)?

**Comment by LordZombi** [comment]
**tldr;**

"Hello {'@'} World" instead "Hello @ World"

**Comment by Zopt** [comment]
> **tldr;**
> 
> "Hello {'@'} World" instead "Hello @ World"

Good way
